src/app/zap-templates/zcl/data-model/silabs/general.xml

```diff
@@ -556,4 +556,9 @@ limitations under the License.
     <attribute side="server" code="0x006F" define="STATUS_FLAGS" type="BITMAP8" min="0x00" max="0x0F" writable="false" default="0x00" reportable="true" optional="false">status flags</attribute>
     <attribute side="server" code="0x0100" define="APPLICATION_TYPE" type="INT32U" min="0x00000000" max="0xFFFFFFFF" writable="false" optional="true">application type</attribute>
   </cluster>
+  <bitmap name="LevelControlFeature" type="BITMAP32">
+      <field name="OnOff" mask="0x1"/>
```

Comment:
Should have a `<cluster>` tag here with the right code="..." bit so it will actually be attached to this cluster and codegen for it will happen properly.

Reply:
I actually took examples on other bitmap for the featuremaps of other clusters that are done and they are never wrapped by a cluster tag. I believe I did put it in inside the cluster the first time while working on it and it wasn't generated correctly. 

Any ideas? I will give it another try

Reply:
Not wrapped in a cluster tag.  You want:
```
  <bitmap name="LevelControlFeature" type="BITMAP32">
      <cluster code="0x0008"/>
      <field name="OnOff" mask="0x1"/>
      etc
```
all outside the `<cluster>` tag that defines the cluster